hwtracer/build.rs

```diff
@@ -100,8 +100,9 @@ fn main() {
 
     // Generate a `compile_commands.json` database for clangd.
     let ccg = CompletionWrapper::new("hwtracer", &env::var("CARGO_MANIFEST_DIR").unwrap());
-    env::set_var.call(ccg.build_env());
-    env::set_var("YK_COMPILER_PATH", ykllvm_bin("clang"));
```

Comment:
Just checking: `YK_COMPILER_PATH` isn't needed anymore? Is that because it's an arg to the wrapper now?

Reply:
It's now defined here https://github.com/ykjit/yk/pull/697/files#diff-974a79e2a0deb1dfad4350c727a981fb2ea2df6c09e874832df6d2048db5206aR79